mappings/net/minecraft/world/gen/feature/DefaultBiomeFeatures.mapping

```diff
@@ -173,3 +173,9 @@ CLASS net/minecraft/class_3864 net/minecraft/world/gen/feature/DefaultBiomeFeatu
 		ARG 0 builder
 	METHOD method_39420 addFrozenLavaSpring (Lnet/minecraft/class_5485$class_5495;)V
 		ARG 0 builder
+	METHOD method_39931 addMelon (Lnet/minecraft/class_5485$class_5495;)V
```

Comment:
```suggestion
	METHOD method_39931 addMelons (Lnet/minecraft/class_5485$class_5495;)V
```

Reply:
```suggestion
	METHOD method_39931 addMelonPatches (Lnet/minecraft/class_5485$class_5495;)V
```
would probably fit best

Reply:
Currently patches are not referred to in these methods, but I am fine with this if the whole class is looked at as well.